pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -6,16 +6,17 @@
 Package v1alpha1 configuration file contains all the options available for configuring a machine.
 
 To generate a set of basic configuration files, run:
-```bash
-talosctl gen config --version v1alpha1 <cluster name> <cluster endpoint>
-````
+
```

Comment:
This line is actually rendered on our configuration reference page as code block.
Was it something related to deepcopy-gen?

Reply:
Ah, I tried to fix the godoc output, I will double check it still renders as code block in the docs.

Reply:
yep, it looks good in our docs now and in godoc as well